test/Riok.Mapperly.Tests/Mapping/ObjectPropertyFlatteningTest.cs

```diff
@@ -278,6 +521,28 @@ public void ManualUnflattenedProperty()
             );
     }
 
+    [Fact]
+    public void ManualUnflattenedPropertyShouldIgnoreAuto()
```

Comment:
Not necessarily, the first one (`ManualUnflattenedPropertyShouldIgnoreAuto`) tests whether the final configuration selects the manually-suggested member, overriding the automatic configuration, while the second one (`AutoUnflattenedManuallyConfiguredPropertyShouldOnlySetOnce`) tests whether the final configuration, provided a manual override, doesn't assign the same member twice, it being the one that automatic configuration would have otherwise chosen.